src/models/user.js

```diff
@@ -4,6 +4,7 @@ const generateFullName = require('./helpers/generateFullName');
 
 const featureFlags = [
   'recipient_goals_objectives',
+  'resources_dashboard',
 ];
```

Comment:
I guess I need to write a jira to remove recipient_goals _objectives

Reply:
Do you want me to remove this line in this PR?

Reply:
My preference is to keep it separate - I'll write a ticket now

Reply:
for ref: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1255